add allure report

Sergej-Skom opened the pull request.

Sergej-Skom committed:
add allure report

pomegranate282922 reviewed (commented):
Привет! Судя по всему не все коммиты попали в пулл реквест. Перепроверь, пожалуйста

Sergej-Skom committed:
Добавлен отчет Allure

Sergej-Skom committed:
Коды всех классов добавлены

Sergej-Skom committed:
Добавлен отчет Allure

pomegranate282922 commented on src/main/java/org/example/Courier.java:
```diff
@@ -0,0 +1,45 @@
+
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
```
Можно использовать библиотеку lombok для уменьшения boilerplate кода 
Можно улучшить

pomegranate282922 commented on src/main/java/org/example/Main.java:
```diff
@@ -0,0 +1,17 @@
+
+//TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
+// click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
+public class Main {
```
Этот класс нужно удалить 
Нужно исправить

pomegranate282922 commented on src/test/java/TestOrderList.java:
```diff
@@ -0,0 +1,30 @@
+    @Description("Список заказов получен успешно")
+    public void checkOrderList(){
+        apiOrder.getOrderList()
+                .then().assertThat().body("orders", notNullValue())
```
Лучше проверить не пустоту списка 
Можно улучшить

pomegranate282922 commented on src/test/java/api/ApiConfiguration.java:
```diff
@@ -0,0 +1,19 @@
+    public void setupRequestSpecification()
+    {
+        requestSpecification = RestAssured.given()
+                .baseUri("http://qa-scooter.praktikum-services.ru/")
```
Все урлы выносим как константы 
Нужно исправить

pomegranate282922 reviewed (commented):
Отличная работа! Все тесты реализованы. Применена параметризация. Везде есть удаление тестовых данных. Есть аллюр описания шагов и тестов.
Замечания: по мелочи

Sergej-Skom committed:
add allure report

Sergej-Skom committed:
Класс Main удалил
 lombok в классе Courier использовал
 Урлы вынес как константы, создал для них класс ApiEndpoints

pomegranate282922 reviewed (approved):
Отлично, зачет!
